### ISSUE TITLE
The environment of Point-NN

### ISSUE THREAD

**Issue by whuhxb** [ZrrSkywalker/Point-NN#1]
Hi @ZrrSkywalker 

What's the specific environment of Point-NN? Which cuda version and pytorch version? Thanks a lot.

**Comment by ZrrSkywalker** [comment]
Hi. Thanks for your interest. 
Our environment is:
```bash
torch             1.7.1+cu110
torchvision       0.8.2+cu110
cudatoolkit       10.2.89
```
Our nvcc -V with RTX 3090 is
```bash
cuda_11.2
```

**Comment by whuhxb** [comment]
> Hi. Thanks for your interest. Our environment is:
> 
> ```shell
> torch             1.7.1+cu110
> torchvision       0.8.2+cu110
> cudatoolkit       10.2.89
> ```
> 
> Our nvcc -V with RTX 3090 is
> 
> ```shell
> cuda_11.2
> ```

Could other environment work? For example, cuda10.1 and pytorch 1.7.0. Thanks a lot.

**Comment by ZrrSkywalker** [comment]
Yes it can work.

**Comment by whuhxb** [comment]
> Yes it can work.

OK. Will have a try. Thanks.

**Comment by whuhxb** [comment]
> Yes it can work.

Have you tested this backbone for semantic segmentation task? Thanks.

**Comment by whuhxb** [comment]
> Yes it can work.

The code runs so quick. How to visualize the classification result? And, how to output the classification result? Thanks.

**Comment by ZrrSkywalker** [comment]
> > Yes it can work.
> 
> The code runs so quick. How to visualize the classification result? And, how to output the classification result? Thanks.

The classification logits of the test set are here: https://github.com/ZrrSkywalker/Point-NN/blob/9707b2ed5afe79be79b9a19d70aec9b04d8fb648/run_nn_cls.py#L120

but you need to use the `best_gamma` after the loop to obtain final logits of Point-NN.

**Comment by whuhxb** [comment]
> > > Yes it can work.
> > 
> > 
> > The code runs so quick. How to visualize the classification result? And, how to output the classification result? Thanks.
> 
> The classification logits of the test set are here:
> 
> https://github.com/ZrrSkywalker/Point-NN/blob/9707b2ed5afe79be79b9a19d70aec9b04d8fb648/run_nn_cls.py#L120
> 
> but you need to use the `best_gamma` after the loop to obtain final logits of Point-NN.

OK. Thanks.
